src/factory/src/state.rs

```diff
@@ -53,11 +61,17 @@ impl Default for State {
 }
 
 pub fn get_token_bytecode() -> Vec<u8> {
-    State::get()
-        .borrow()
-        .token_wasm
-        .clone()
-        .expect("the token bytecode should be set before accessing it")
+    #[cfg(target_family = "wasm")]
+    {
+        State::get()
+            .borrow()
+            .token_wasm
+            .clone()
+            .expect("the token bytecode should be set before accessing it")
+    }
+    // In the mocking context we don't care about injecting wasm code anywhere
+    #[cfg(not(target_family = "wasm"))]
+    vec![]
```

Comment:
I don't see why this logic is needed. Having this method return real value doesn't hurt testability in any way. So why change the logic for tests?

Reply:
It panics because no bytecode was set from the manager mock

Reply:
So why not set the wasm to vec![] when mocking? It's better then writing test logic in the main code.